pkg/controller/build/controller_test.go

```diff
@@ -74,6 +74,29 @@ func newBuild(name string) *v1alpha1.Build {
 	}
 }
 
+func (f *fixture) testSetup() error {
+	//t.Helper()
```

Comment:
That could be `f.t.Helper()` I think. Not really sure `fixture` should hold the `t` reference but that's not this PR concern anyway :wink: